### PR TITLE
test: remove tests for unimplemented features

### PR DESCRIPTION
**Pull request by rmocanu-ionos**

### CODE REVIEW

File: functional/FunctionalTest.java [hunk 1]
```diff
@@ -2026,38 +2026,38 @@ public static void copyObject() throws Exception {
...
-        "[SSE-S3]",
-        sseS3,
-        CopyObjectArgs.builder()
-            .bucket(bucketName)
-            .object(getRandomName())
-            .sse(sseS3)
-            .source(CopySource.builder().bucket(getRandomName()).object(getRandomName()).build())
-            .build(),
-        false);
+    // testCopyObject(
```
Comment (digna-ionos) [inline functional/FunctionalTest.java:2029]: Why it is commented out?

Comment (rmocanu-ionos) [reply functional/FunctionalTest.java:2029]: I was thinking that maybe this will be added to the API at some point and having the test would help us then

Comment (digna-ionos) [reply functional/FunctionalTest.java:2029]: I think that the CopyObject API is working properly.

Comment (rmocanu-ionos) [reply functional/FunctionalTest.java:2029]: oh, this test was causing a 405

Comment (digna-ionos) [reply functional/FunctionalTest.java:2029]: i see that is with sse. Maybe that's why

**Review by digna-ionos — APPROVED** [review]

**Pull request merged**